app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -42,18 +47,38 @@ const NoteForm = ({ onClose, refetch, note, isEdit }) => {
               label="Title"
               name="title"
             />
-            <Textarea
+            <Input
               required
               className="w-full flex-grow-0"
               label="Description"
               name="description"
-              rows={8}
+            />
+            <Select
+              defaultMenuIsClose
+              isSearchable
+              required
+              className="w-full flex-grow-0"
+              label="Assigned Contacts"
+              name="assignedContacts"
+              options={CONTACTS_DROPDOWN_DATA}
+              placeholder="Select Role"
+            />
+            <Select
+              defaultMenuIsClose
+              isMulti
+              required
+              className="w-full flex-grow-0"
+              label="Tags"
+              name="tags"
+              options={TAGS_DROPDOWN_DATA}
+              placeholder="Select Role"
```

Comment:
The label says `Tags`. But the placeholder says `Role`. Pay attention to the keywords.